src/dom.rs

```diff
@@ -606,6 +608,33 @@ impl<A> DomBuilder<A> where A: AsRef<Node> {
     }
 }
 
+impl<A> DomBuilder<A> where A: AsRef<Node> {
+    #[inline]
+    pub fn child_signal<B>(mut self, signal: B) -> Self
+        where B: Signal<Item = Dom> + 'static {
+
+        assert_eq!(self.has_children, false);
+        self.has_children = true;
+
+        let element = self.element.as_ref().clone();
+        let callbacks = &mut self.callbacks;
+        let state: Rc<RefCell<Option<Dom>>> = Rc::new(RefCell::new(None));
```

Comment:
`operations::insert_children_signal_vec()` uses an `Arc<Mutex<T>>`, but since there's no multithreading I think `Rc<RefCell<T>>` should work fine too?